src/instrumentsscene/view/instrumentlistmodel.cpp

```diff
@@ -395,6 +397,16 @@ bool InstrumentListModel::hasSelection() const
     return m_selection->hasSelection();
 }
 
+QString InstrumentListModel::selectedInstrumentDescription() const
+{
+    QList<int> selectedRows = m_selection->selectedRows();
+    if (selectedRows.length() != 1) {
+        return QString();
+    }
+    CombinedInstrument instrument = m_instruments.at(selectedRows.at(0));
+    return instrument.templates.at(instrument.currentTemplateIndex)->description;
```

Comment:
I guess this is the place where you'd need to pick on the translation

Reply:
Probably